src/modules/submissions.js

```diff
@@ -533,9 +551,14 @@
 			] );
 		} else {
 			// If not yet accepted, comment out cats
-			text = text.replace( /\[\[Category:/gi, '[[:Category:' );
+			text = text.replace( /\[\[:?Category:(.*?\s*)\]\]/gi, function call( match, category ) {
+				if ( draftArticleSubcats.includes( category ) ) {
+					return '[[Category:' + category + ']]';
+				} else {
+					return '[[:Category:' + category + ']]';
+				}
+			} );
 		}
-
```

Comment:
nitpick: here and further down, there are a bunch of removed lines; please add them back in so the diff is smaller (ideally, the diff is as small as possible)

Reply:
Did you mean the blank lines which were removed?

Reply:
Yes, the blank lines